.editorconfig

```diff
@@ -0,0 +1,9 @@
+[*]
```

Comment:
@pbredenberg aren't we usually making this a symlink to the file in the standardization repo (i.e. `node_modules/@silvermine/standardization/.editorconfig`) so that it's not duplicated everywhere?

Reply:
did it! @jthomerson back to you